Src/FluentAssertions/Equivalency/Steps/GenericDictionaryEquivalencyStep.cs

```diff
@@ -57,8 +78,9 @@ private static bool AssertSubjectIsNotNull(object subject)
         if (actualDictionary is null)
         {
             AssertionScope.Current.FailWith(
-                $"Expected {{context:subject}} to be a dictionary or collection of key-value pairs that is keyed to type {expectedDictionary.Key}. " +
-                $"It implements {actualDictionary}.");
+                "Expected {context:subject} to be a dictionary or collection of key-value pairs that is keyed to " +
+                $"type {expectedDictionary.Key}. " +
+                "It implements <null>.");
```

Comment:
How does something implement null? :upside_down_face: 
Let's either remove this line or e.g. print the all `IDictionary<,>` interfaces implemented by `subject` like it did before #1603.
I'm fine with either.

Reply:
I will remove it then.